frontend/src/pages/list/list.rs

```diff
@@ -128,16 +131,16 @@ pub fn view(model: &Model, ctx: &Context) -> Node<Msg> {
                 div![C!["panel-body"],
                     div![
                         C!["form-group"],
-                        label![attrs!{At::For => "list-create-id"}, "#"],
+                        label![attrs!{At::For => "list-create-id"}, "Id"],
                         span![C!["form-control"], attrs!{At::Id => "list-create-id"}, "Auto"],
                     ],
                     div![
                         C!["form-group"],
-                        label![attrs!{At::For => "list-create-title"}, "Id"],
+                        label![attrs!{At::For => "list-create-description"}, "Description"],
```

Comment:
Stored Lists don't seem to have a title in the model, was this just for debug purposes or does it have some purpose I'm not getting? Changed to Description for now, as that's a field that the model contains.